Unzip-files-s3/unzip_s3.py

```diff
@@ -0,0 +1,89 @@
+import boto3
+import io
+import zipfile
+import re
+from urllib.parse import unquote_plus
+
+# Initialize S3 client outside the handler for connection reuse
+s3_client = boto3.client('s3')
+
+def extract_key_from_event(event):
+    """
+    Extract the S3 bucket name and object key from the Lambda event.
+    """
+    record = event['Records'][0]['s3']
+    bucket_name = record['bucket']['name']
+    object_key = unquote_plus(record['object']['key'])
+    return bucket_name, object_key
+
+def clean_filename(filename):
+    """
+    Cleans the filename to be URL-friendly.
+    """
+    return re.sub(r'[~ñé]', '', filename)
```

Comment:
Hi @aliraza556 the characters were just an example, we need all "strange" characters removed, any accents, tildes, etc. Can we implement something like here where we specify characters that we want and everything else is replaced by dashes? https://www.himpfen.com/generate-seo-friendly-urls-blog-posts-python/